### DISCUSSION TITLE
Daily 29/9

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#44] [General]
Ayer agregué más historias de usuario correspondientes a cada una de las páginas que propuse en el mapa del sitio, también hice una nueva versión del ERS con las restricciones que agregaron a la Wiki. Todavía faltan personas para agregarse al documento pero el contenido estaría bastante avanzado. Hoy planeo seguir convirtiendo las historias de usuario en issues y tal vez tomar algunas de las tareas de diseño de páginas como para que para el final de este sprint ya tengamos todos los diseños finalizados. También espero estar para la clase de SGC.

**Comment by NataliaAlvarezIspc** [comment]
Ayer leí el nuevo capítulo de SGC y estuve repasando CSS. Además completé algunas partes del ERS.
Hoy hice la nueva  lección de Inglés. Participaré de las clases de SGC y de Full Stack. Además continuaré profundizando CSS.

**Comment by rocappellano** [comment]
Repasaré teórico, de HTML y los demás temas para la recolección de saberes de este fin de semana

**Comment by IngridAlcocer05** [comment]
Ayer realice las actividades de CMS. Hoy continúe con el modulo de Python.

**Comment by AhumadaCeleste** [comment]
Ayer termine de estudiar pare el examen de full stack. Hoy completare la actividad de cms y le lección de ingles de la semana. Mañana retomare python
